### ISSUE TITLE
Chest upgrade hover hard to see

### ISSUE THREAD

**Issue by confuser** [Hex27/Kingdoms#14]
In the client, hovering over the chest upgrade makes most of the tooltip go off screen making it very difficult to see what it is.

Solutions are to either move it over by one inventory location wise, or better yet, split the lore over multiple lines.

I would love to send some PRs to fix a lot of these things, but the code seems a little difficult to follow in its current state.

**Comment by sgdc3** [comment]
yesterday I sent a PR with an entire code cleanup, but we need to compact all the if else conditions and split the main file into more class files and functions...

**Comment by confuser** [comment]
Well, the formatting is quite inconsistent on all the files too, but it'd be good to separate each of the sub commands into their own classes.

**Comment by sgdc3** [comment]
Yes, yesterday I spent 2 hours only to fix the code indentation and simplify some weird aggregated if :/

**Comment by confuser** [comment]
I have some free time this morning, so I'll see how much I can get done :)

**Comment by sgdc3** [comment]
This project has a good potential, but the development is slow and not optimized.... :/

**Comment by Hex27** [comment]
Its one of the first few major plugins that I have made, and one of my messiest. I'm not so sure how to fix it beyond this point

**Comment by sgdc3** [comment]
@confuser Can you read my PM on the spigot forum? :) Thanks
